Automattic/pocket-casts-android
# Onboarding: Add import screens

| 📘 Project: #524  |
| --- |

## Description

This adds the import screens to the onboarding flow. 

This PR does not hook up the import button on the recommendations screen, that will be done in a follow-up PR that is making additional changes to the recommendations screen.

I found it surprisingly tricky to create a nice numbered list using Jetpack compose. I feel like there should be an easier way than manually positioning the text using a `ConstraintLayout` like I am here. If you have any better ideas, let me know.

I'm importing the icons as pngs because the svgs were not compatible with Android.

## Testing Instructions

1. Get to the welcome screen of the onboarding flow (requires enabling the feature flag in `base.gradle`, and you may want to update [this line](https://github.com/Automattic/pocket-casts-android/blob/f33cf2b1dd34a857e9861a67041a9bfcaed1c023/modules/features/account/src/main/java/au/com/shiftyjelly/pocketcasts/account/onboarding/OnboardingFlowComposable.kt#L50) to just start at `OnboardingNavRoute.welcome`).
2. Tap on the "Import your podcasts" card
3. Observe the Import screen is loaded with 2 options: "Import from Castbox" and "Import from other apps"
4. Tap on each option and follow the steps to import podcasts from some other app and make sure the steps are clear enough to follow
5. From the "Import from Casbox" screen, check that there is an "Open Castbox..." button if you have Castbox installed, and that the button is removed if you do not have Castbox installed.

## Screenshots or Screencast 

https://user-images.githubusercontent.com/4656348/206013585-15dbf868-6a8f-4164-8743-044b92dd490a.mov

## Checklist
 
#### I have tested any UI changes...
